test/native/testRunner.hpp

```diff
@@ -68,6 +68,8 @@ struct TestCase {
 
 #define __ASSERT_OR_CHECK_OP(isAssert, val1, op, val2)                                                               \
     {                                                                                                                \
+        _Pragma("GCC diagnostic push");                                                                              \
+        _Pragma("GCC diagnostic ignored \"-Waddress\"");                                                             \
```

Comment:
This is getting more and more intricate.
Would not it be better to have a special version of macros for strings?

Reply:
We could split it at some point.